contract/models/res_partner.py

```diff
@@ -14,13 +14,14 @@ class ResPartner(models.Model):
     )
 
     def _compute_contract_count(self):
-        Contract = self.env['account.analytic.account']
-        today = fields.Date.today()
+        contract = self.env['account.analytic.account']
```

Comment:
Please don't modify this variable name which is correct.

Reply:
Not according to the guidelines that say to use lowercase for variable names: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id34

Reply:
Yeah, that's the only exception that is not explicitly written, but a real convention, even on Odoo core: as this represent the handler of the model, it's very similar in concept to python classes, and thus, they are put in upper case.

If you don't want to put it in upper, then you must use a meaningful variable name, like `contract_obj` or `contract_model`, which are longer, which is the reason for the other naming style, but not `contract`, as this can be interpreted as a record of that model.

Reply:
And the most important thing: if you are not touching that line, better to leave it as is for reducing the diff, even if it's not exactly your style. Note that Travis was green even with this variable name, as it's not checked due to the reasons above.